volatility3/framework/plugins/windows/psxview.py

```diff
@@ -0,0 +1,158 @@
+import datetime
+from typing import List, Iterable, Set, Dict
+from volatility3.framework import renderers, interfaces
+from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces import plugins
+from volatility3.plugins.windows import pslist, psscan
+
+class PsXview(plugins.PluginInterface):
+    """Find hidden processes with various process listings"""
+    _required_framework_version = (2, 0, 0)
+
+    @classmethod
+    def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
+        return [
+            requirements.TranslationLayerRequirement(name='primary',
```

Comment:
We tend to use a `ModuleRequirement` these days, instead of separate `TranslationLayerRequirement`s and `SymbolTableRequirement`s.